versatileimagefield/files.py

```diff
@@ -99,4 +107,8 @@ def __get__(self, instance=None, owner=None):
                 file.ppoi = ppoi
 
         # That was fun, wasn't it?
+        # Finally, ensure all the sizers/filters are available after pickling
```

Comment:
Django slightly changed the implementation of the FileDescriptor after Django 1.8 (I believe this implementation is based on the code that was in Django 1.8)

https://github.com/django/django/blob/3.2rc1/django/db/models/fields/files.py#L158-L210

Django 3.2rc1 also includes a change to the base class of FileDescriptor and has changed the use of `name` to `attname` in some places

https://github.com/django/django/commit/a93425a37f4defdb31d4ca96bb3bf6da21f0b5ce
https://github.com/django/django/commit/6599608c4d0befdcb820ddccce55f183f247ae4f